src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -275,6 +275,16 @@ export class CalciteColorPicker {
 
   @State() savedColors: string[] = [];
 
+  @State() scopeTop: number;
+
+  @State() scopeLeft: number;
+
+  @State() scopeOrientation: "vertical" | "horizontal";
+
+  @State() hueScopeLeft: number;
```

Comment:
Nitpick: left and top props seem to be updated and used at the same time. Maybe these could be consolidated.